src/__experimental__/components/switch/switch-slider.component.js

```diff
@@ -17,9 +18,10 @@ const SwitchSlider = (props) => {
     info,
     useValidationIcon,
   } = props;
+  const t = useTranslation();
 
-  const on = I18n.t("switch.on", { defaultValue: "ON" }).toUpperCase();
-  const off = I18n.t("switch.off", { defaultValue: "OFF" }).toUpperCase();
+  const on = t("switch.on", "ON").toUpperCase();
```

Comment:
Is this going to work if someone uses `Switch` component that is not wrapped by the `I18nProvider`?

Reply:
No, then `I18nProvider` will throw an error.

Reply:
I'm not sure if it's a good idea to have a component library that enforces the use of `I18nProvider`.